biojava-core/src/test/java/org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.java

```diff
@@ -0,0 +1,47 @@
+package org.biojava.nbio.core.sequence.io.embl;
```

Comment:
All source code in biojava requires the license header

Reply:
done.